targets/TARGET_Cypress/scripts/mbed_set_post_build_cypress.cmake

```diff
@@ -3,48 +3,56 @@
 
 include(mbed_set_post_build)
 
-set(MBED_POST_BUILD_CYPRESS_DIR "${CMAKE_CURRENT_LIST_DIR}")
-
 #
 # Merge Cortex-M4 HEX and a Cortex-M0 HEX.
 #
-function(mbed_post_build_psoc6_merge_hex mbed_target_name)
-    find_package(Python3)
+macro(mbed_post_build_psoc6_merge_hex mbed_target_name)
+    if("${mbed_target_name}" STREQUAL "${MBED_TARGET}")
```

Comment:
Here it's `mbed_target_name`. What about `mbed_psoc_target`?

Reply:
Okay, I can do that